pkgs/applications/graphics/oculante/default.nix

```diff
@@ -44,19 +46,23 @@ rustPlatform.buildRustPackage rec {
   buildInputs = [
     openssl
     fontconfig
-  ] ++ lib.optionals stdenv.isLinux [
+   ] ++ lib.optionals stdenv.isLinux [
+    libGL
+    libX11
```

Comment:
```suggestion
```

Reply:
I think you missed this :)

Reply:
ugh, finally got it, I don't know how I missed that, I swear I fixed it like three times. :upside_down_face:

Reply:
hmmm... did you forget to push?

Reply:
I'm scared to even say, maybe this time?

Reply:
I got you: [`b8d9c8a` (#229532)](https://github.com/NixOS/nixpkgs/pull/229532/commits/b8d9c8a7efab3186bd55b95d44781b67f9714ac4)